lib/ansible/modules/windows/win_cert_stat.ps1

```diff
@@ -0,0 +1,103 @@
+#!powershell
+
+# Copyright: (c) 2019, Micah Hunsberger
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+
+function ConvertTo-Timestamp($start_date, $end_date) {
+    if ($start_date -and $end_date) {
+        return (New-TimeSpan -Start $start_date -End $end_date).TotalSeconds
+    }
+}
+
+function Format-Date([DateTime]$date)
+{
+    return $date.ToUniversalTime().ToString('yyyy-MM-ddTHH:mm:ssK')
```

Comment:
You can also do `ToString("o")` which I believe adds the UTC offset.

Reply:
It does, but it also adds unnecessary precision, and will format it as "local time" + "utc offset", instead of converting it to UTC and noting that it is UTC time.
```
(Get-Date).ToString('o') # 2019-12-04T07:55:28.7763737-08:00
(Get-Date).ToUniversalTime().ToString('yyyy-MM-ddTHH:mm:ssK') # 20119-12-04T15:55:28Z
```
It adds all the partial seconds even when the date-time object has them set to 0
```
(Get-Date -Millisecond 0 -Second 0).tostring('o') # 2019-12-04T11:08:00.0000000-05:00
```
which is why I opted for the custom format string, since most (if not all?) of certificate expiration dates only have a precision of seconds, not milliseconds

Reply:
Fair enough makes sense